app/web/features/search/FilterDialog.tsx

```diff
@@ -212,15 +233,45 @@ export default function FilterDialog({
               />
 
               <Select
-                id="can_host_status_filter"
-                value={hostingStatusFilter}
-                onChange={(e) => {
-                  setHostingStatusFilter(e.target.value as number);
-                }}
+                id="can_host_status_filter_1"
+                variant="outlined"
                 label={t("search:form.host_filters.hosting_status_field_label")}
+                multiple={true}
+                menuItems={true}
+                value={hostingStatusFilter}
+                onChange={(e) =>
+                  setHostingStatusFilter(e.target.value as number[])
+                }
+                input={
+                  <Input
+                    className={classes.inputHostingStatus}
+                    id="select-multiple-chip"
+                  />
+                }
+                className={classes.marginBottom}
+                native={false}
+                renderValue={(selected) => (
+                  <div className={classes.chips}>
+                    {(selected as number[]).map(
+                      (
+                        value: Exclude<
+                          HostingStatus,
+                          | HostingStatus.HOSTING_STATUS_UNKNOWN
+                          | HostingStatus.HOSTING_STATUS_UNSPECIFIED
+                        >
+                      ) => (
+                        <Chip
+                          key={value}
+                          label={getHostingStatusOptions(t)[value]}
+                          className={classes.chip}
+                        />
+                      )
+                    )}
+                  </div>
+                )}
+                MenuProps={MenuProps}
                 optionLabelMap={getHostingStatusOptions(t)}
                 options={[
-                  HostingStatus.HOSTING_STATUS_UNSPECIFIED,
                   HostingStatus.HOSTING_STATUS_CAN_HOST,
                   HostingStatus.HOSTING_STATUS_MAYBE,
                   HostingStatus.HOSTING_STATUS_CANT_HOST,
```

Comment:
What if the user is just looking to meet up with someone and doesn't care about their hosting status?

Reply:
then leaves the dropdown unselected, and since it's undefined value isn't sent to the server

Reply:
Ah perfect thanks just wanted to make sure it was possible.